tests/ui/functions-closures/bare-fn-implements-fn-mut.rs

```diff
@@ -1,3 +1,5 @@
+//! Tests that bare functions implement the `FnMut` trait correctly.
+
 //@ run-pass
 
 fn call_f<F:FnMut()>(mut f: F) {
```

Comment:
Suggestion: can you reformat this file (you can run nightly rustfmt against this test path)? It has some unusual formatting that's leftover from 11 years ago that's a bit distracting.